src/main/webapp/table-manager.js

```diff
@@ -12,6 +17,8 @@ class TableManager {
     this.pageNumber = document.getElementById('page-number');
     this.pageSizeSelect = document.getElementById('page-size-select');
```

Comment:
considering making these const named variables

Reply:
I am going to leave these as-is because they only appear in one place (here) and the semantics of the `getElementById` function makes their purpose clear.  Should they ever need to be used in multiple places in a future update, I will make them const named variables.